x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/assets/assets.tsx

```diff
@@ -36,8 +37,12 @@ interface AssetsPanelProps {
 
 export const AssetsPage = ({ packageInfo }: AssetsPanelProps) => {
   const { name, version } = packageInfo;
+  const { spaces } = useKibana<StartPlugins>().services;
   const pkgkey = `${name}-${version}`;
-
+  const assetInstallSpaceId =
```

Comment:
nitpick: can we declare this in the useEffect L65 as it seems not used elsewhere?